packages/components/button/src/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { cx } from 'emotion';
+import { CommonProps } from '@contentful/f36-core';
+import { IconComponent } from '@contentful/f36-icon';
+import { Button } from '../Button';
+import getStyles from './ToggleButton.styles';
+
+export interface ToggleButtonProps extends CommonProps {
```

Comment:
shouldn't it just extend ButtonProps?

Reply:
Actually was doing some testing, and I think it's better to have it extending only the `CommonProps`, since the toggle button should use the variants, or have different sizes. Extending the `ButtonProps` we would make it so those other props also be typed onto the `ToggleButton`

Reply:
Ok, if it makes sense more